mods/stairs/init.lua

```diff
@@ -256,355 +373,425 @@ if replace then
 end
 
 
--- Stair/slab registration function.
--- Nodes will be called stairs:{stair,slab}_<subname>
+-- Stair/slab/stairi/stairo registration function.
+-- Nodes will be called stairs:{stair,slab,stairi,stairo}_<subname>
 
-function stairs.register_stair_and_slab(subname, recipeitem,
```

Comment:
In order to add backward compatibility for a function, define `mod.old_function = mod.new_function` after the `new_function` is registered. In this case:
```lua
stairs.register_stair_and_slab = stairs.register_stair_and_slab_and_stairi_and_stairo
```